docs/section19/Rule 19-39.md

```diff
@@ -0,0 +1,84 @@
+
+        - Add each zone's OA CFM schedule to the list of zone OA CFM schedules for the baseline: `zone_OA_CFM_list_of_schedules_b[counter] = get_min_OA_CFM_sch_zone(B_RMI, zone.id)`  
+        - Get the analogous zone object from the proposed RMI (if this is not strictly the correct syntax please advise Weili): `zone_p = get_component_by_id(P_RMI,match_data_element(P_RMI,Zone,zone.id))`    
+        - Add each zone's OA CFM schedule to the list of zone OA CFM schedules for the proposed: `zone_OA_CFM_list_of_schedules_p[counter] = get_min_OA_CFM_sch_zone(P_RMI, zone_p.id)`  
+        - Add to counter: `counter = counter + 1`
+
+        - Create aggregated OA CFM schedule for the baseline for the zones served by the hvac system: `aggregated_min_OA_schedule_across_zones_b = aggregate_min_OA_schedule_across_zones(B_RMI,zone_OA_CFM_list_of_schedules_b)`  
+        - Create aggregated OA CFM schedule for the proposed for the zones served by the baseline hvac system: `aggregated_min_OA_schedule_across_zones_p = aggregate_min_OA_schedule_across_zones(P_RMI,zone_OA_CFM_list_of_schedules_p)`  
+        - Reset OA_CFM_schedules_match boolean variable to True: `OA_CFM_schedules_match = True`  
+        - Check if schedules mismatch for any hours of the year, if so set boolean to false: `for p to Range(8760):`  
+            - Check if the baseline and proposed schedules mismatch: `if aggregated_min_OA_schedule_across_zones_b[p] != aggregated_min_OA_schedule_across_zones_p[p]: OA_CFM_schedule_match = false`   
+        - Sum OA CFM across baseline schedule to get the total for comparison purposes in the rule assertions: `modeled_baseline_total_zone_min_OA_CFM = sum(aggregated_min_OA_schedule_across_zones_b)`  
+        - Sum OA CFM across proposed schedule to get the total for comparison purposes in the rule assertions: `modeled_proposed_total_zone_min_OA_CFM = sum(aggregated_min_OA_schedule_across_zones_p)`  
+
+        - Get the maximum_outdoor_airflow for the fan system: `max_OA_flow = hvac.fan_system.maximum_outdoor_airflow`  
+        - Get the design supply air flow: `design_supply_flow = list(get_fan_system_object_supply_return_exhaust_relief_total_power_flow(B_RMI, hvac.fan_system)["supply_fans_airflow])`  
+        - Get the minimum fan system air flow: `min_air_flow = hvac.fan_system.minimum_airflow`  
+        - Get the minimum fan system OA flow: `min_OA_flow = hvac.fan_system.minimum_outdoor_airflow` 
+        - Check if it was modeled as a 100% outdoor air unit: `if max_OA_flow == min_OA_flow and min_OA_flow == design_supply_flow: modeled_100_percent_OA = true`  
+        - Check if it possbily may have been modeled as a 100% outdoor air unit: `if if max_OA_flow == design_supply_flow and min_OA_flow == min_air_flow: possibly_modeled_100_percent_OA = true`  
+        - Check if it is NOT likely that it was not modeled as a 100% outdoor air unit: `if min_OA_flow != min_air_flow: likely_NOT_modeled_100_percent_OA = true`  
```

Comment:
Maybe if should be if min_OA_flow < min_air_flow: likely_NOT_modeled_100_percent_OA = true. Also, in this case, wouldn't it be certainly not a DOAS?